pkg/kustomize/interface.go

```diff
@@ -3,4 +3,7 @@ package kustomize
 // Kustomizer defines common kustomize actions used within Jenkins X
 type Kustomizer interface {
 	Version(extraArgs ...string) (string, error)
+	HasKustomize() bool
```

Comment:
_HasKustomizeConfig_ or _UsesKustomize_ or _ContainsKustomizeConfig_ might be better.

Reply:
Also, you might want to pass the directory from which to search the search as a parameter. This way you don't need to pass it to the constructor.